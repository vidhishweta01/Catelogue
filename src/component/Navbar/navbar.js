import React from 'react';
import { NavLink } from 'react-router-dom';
import styles from './navbar.module.css';

const Navbar = () => (
  <nav className={styles.navbar}>
    <NavLink className={styles.links} to="/">Pokemon</NavLink>
    <NavLink className={styles.links} to="/pokelist">Pokemon list</NavLink>
  </nav>
);

export default Navbar;
