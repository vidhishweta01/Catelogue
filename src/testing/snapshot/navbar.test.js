import React from 'react';
import { BrowserRouter } from 'react-router-dom';
import renderer from 'react-test-renderer';
import Navbar from '../../component/Navbar';

describe('Navbar component', () => {
  it('matches the snapshot', () => {
    const navbar = renderer.create(<BrowserRouter><Navbar /></BrowserRouter>).toJSON();
    expect(navbar).toMatchSnapshot();
  });
});
