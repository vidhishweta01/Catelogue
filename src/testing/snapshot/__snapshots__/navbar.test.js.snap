// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navbar component matches the snapshot 1`] = `
<nav
  className="navbar"
>
  <a
    aria-current="page"
    className="links active"
    href="/"
    onClick={[Function]}
    style={Object {}}
  >
    Pokemon
  </a>
  <a
    aria-current={null}
    className="links"
    href="/pokelist"
    onClick={[Function]}
  >
    Pokemon list
  </a>
</nav>
`;
